openfisca_us_data/datasets/cps/cps.py

```diff
@@ -0,0 +1,99 @@
+from openfisca_us_data.utils import US, dataset
+from openfisca_us_data.datasets.cps.raw_cps import RawCPS
+from pandas import DataFrame, Series
+import h5py
+
+
+@dataset
+class CPS:
+    name = "cps"
+    model = US
+
+    def generate(year: int) -> None:
+        """Generates the CPS dataset.
+
+        Args:
+            year (int): The year of the raw CPS to use.
+        """
+
+        # Prepare raw CPS tables
+        year = int(year)
+        if year not in RawCPS.years:
+            RawCPS.generate(year)
+
+        raw_data = RawCPS.load(year)
+        cps = h5py.File(CPS.file(year), mode="w")
+
+        person, family, household = [
+            raw_data[entity] for entity in ("person", "family", "household")
+        ]
+
+        add_ID_variables(cps, person, family, household)
+        add_basic_income_variables(cps, person)
```

Comment:
Can you change the name of this function?

Reply:
Changed